front/app/modules/commercial/admin_project_templates/index.tsx

```diff
@@ -2,32 +2,21 @@ import React, { ReactNode } from 'react';
 import { ModuleConfiguration } from 'utils/moduleUtils';
 import CreateProjectFromTemplate from './admin/containers/CreateProjectFromTemplate';
 import Tab from './admin/components/Tab';
-import useFeatureFlag from 'hooks/useFeatureFlag';
 import { TTabName } from 'containers/Admin/projects/all/CreateProject';
 import ProjectTemplatePreviewAdminWithEventWrapper from './admin/containers/ProjectTemplatePreviewAdminWithEventWrapper';
+import { RenderOnFeatureFlag } from 'modules/utilComponents';
```

Comment:
Just a suggestion : To keep more consistency, I'd change this to apply both one component per file and consistent folder structure, and to let this one be used in the main app when necessary
```suggestion
import RenderOnFeatureFlag from 'utils/RenderOnFeatureFlag';
```
or 
```suggestion
import RenderOnFeatureFlag from 'components/RenderOnFeatureFlag';
```